ajax/libs/mavo/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "mavo",
+  "version": "0.2.0",
+  "description": "Create web applications with HTML and CSS",
+  "main": "dist/mavo.js",
```

Comment:
As your base path is `dist`, this just needs to be `mavo.js`

Reply:
Did I get it right: I should change it to "main": "mavo.js"?

Reply:
Yup.

Reply:
Thank you! And sorry for all the mistakes I make. 😔 I send you a PR for the first time. I'll do my best next time.